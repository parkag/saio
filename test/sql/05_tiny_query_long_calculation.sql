begin;
CREATE TABLE tenk1 (
    hundred integer,
    thousand integer
);
load 'saio';
set saio_threshold to 1;
set saio_temperature_reduction_factor to 0.99;
explain (costs off) select * from
	tenk1 t1,
	tenk1 t2
where
	t1.thousand = t2.thousand;

rollback;
