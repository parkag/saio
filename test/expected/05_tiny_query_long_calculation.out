begin;
CREATE TABLE tenk1 (
    hundred integer,
    thousand integer
);
load 'saio';
set saio_threshold to 1;
set saio_temperature_reduction_factor to 0.99;
explain (costs off) select * from
	tenk1 t1,
	tenk1 t2
where
	t1.thousand = t2.thousand;
                QUERY PLAN                 
-------------------------------------------
 Merge Join
   Merge Cond: (t1.thousand = t2.thousand)
   ->  Sort
         Sort Key: t1.thousand
         ->  Seq Scan on tenk1 t1
   ->  Sort
         Sort Key: t2.thousand
         ->  Seq Scan on tenk1 t2
(8 rows)

rollback;
